samples/powerpoint/custom-functions/search-wikipedia-api.yaml

```diff
@@ -0,0 +1,221 @@
+order: 1
+id: powerpoint-custom-functions-search-wikipedia-api
+name: Search Wikipedia
+description: Search Wikipedia based on the selected text in the presentation.
+author: cakriwut
+host: POWERPOINT
+api_set: {}
+script:
+    content: |
+        declare var moment: any;
```

Comment:
Unless it's explicitly a JavaScript sample, snippets in this repo try to conform to TypeScript standards. Let's change `var` to `let` or `const` where applicable.

Reply:
Updated